lib/ui/onboarding_screens/screen2.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:paintroid/ui/onboarding_page_app_bar.dart';
+import 'package:paintroid/ui/onboarding_page_bottom_nav_bar.dart';
+
+import '../../workspace/src/ui/drawing_canvas.dart';
+import '../bottom_control_navigation_bar.dart';
```

Comment:
Please check all files for this - imports should not be relative, import as package please